components/fxa-client/ios/FxAClient/FxAccountMetrics.swift

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+public struct MetricsParams {
+    public let parameters: [String: String]
```

Comment:
Same thing here wasn't sure if this best fits as a dictionary or if I should spit out all the expected params